### ISSUE TITLE
ADGUARD HOME - Parental contol seems not working on Android devises.

### ISSUE THREAD

**Issue by ptitnikos** [AdguardTeam/AdGuardHome#5225]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, Other (please mention the version in the description)

### CPU architecture

64-bit ARM

### Installation

Other (please mention in the description)

### Setup

On one machine

### AdGuard Home version

0.107.19

#### What did you do?
I have installed AD Guard Home on a Raspberry PI 4 on  Raspbian GNU/Linux 11 OS.
I have set "use ADGuard parental control web service" and "Use Safe Search" on ADGuard Home settings.
Then i have changed DNS on my ISP router to give IP of my AD Guard as DNS1 on any of my devises (Windows & Android).
 
#### Expected result
Adult content blocked in both cases Search & Typing URL

#### Actual result
On my windows computer, Parental control is working fine and adult content is blocked.
But when i'm trying on my Android smartphone or tab (both in Android 12) it doesn't work. It still displaying adult content web sites when searching of typing the URL.
Any Idea on what I did Wrong ?

**Comment by coanda45** [comment]
I confirm that I have exactly the same problem.
Adguard installed on a synology NAS, DNS changed on the router of my ISP.
Adguard works well on my laptop under debian.
My smartphone under Android 10 (Honor view 10), still display pornographic sites. Same on my wife's smartphone (Samsung 8).

**Comment by coanda45** [comment]
I finally just find the solution.
When you connect to your wireless, you have to change ip, from dynamic to static.
Then change your DNS1 and 2 with the ip of your nas/rasberry.
Just in case, the ip address is written on your AdGuard admin page ==> Installation guid tab ==> click on android.
![image](https://github.com/AdguardTeam/AdGuardHome/assets/108094040/f7d65d13-afe3-4a9a-8f39-7ccd57082f34)

![image](https://github.com/AdguardTeam/AdGuardHome/assets/108094040/bfe3c558-459b-4ce4-a1d0-3cb49554950a)

Of course, If your nas/rasberry is power off, you should not have internet as the dns address will not be available.

**Comment by panagiotis55** [comment]
Hi, I've being trying the parental controls options for a couple of days and have the same exact problem. Cannot find any solution nowhere online. My AGH is installed through docker in a synology nas too.

Steps I've taken:
- In the isp router I've set both the primary and secondary dns pointing to the synology ip.
- I deactivated the private dns setting in android.
- I deactivated the dns through https setting in firefox on android.
- I've disabled the resolving of ipv6 in ADH dns settings.
 Still no results, and this is what the dnscheck.tools website returns when run from android:

> Your` IP addresses:

MEDIANET INVEST AE

94.131.139.126 ns: pri.authdns.ripe.net Athens, Attica, GR

Reserved

192.168.1.20

Your DNS resolvers:

Cloudflare

172.68.61.126 ns: cruz.ns.cloudflare.com Athens, Attica, GR

172.68.61.132 ns: cruz.ns.cloudflare.com Athens, Attica, GR

CLOUDFLARE_2400_CB00_0000_36

2400:cb00:88:1024::ac44:3d7e ns: chloe.ns.cloudflare.com Athens, Attica, GR

2400:cb00:88:1024::ac44:3d84 ns: chloe.ns.cloudflare.com Athens, Attica, GR

INALAN-MNT

45.66.40.13 ptr: 45.66.40.13.inalan.gr Athens, Attica, GR

Oh no! Your DNS responses are not authenticated with DNSSEC:

ECDSA P-256ECDSA P-384Ed25519Valid signaturePASSPASSPASSInvalid signatureFAILFAILFAILExpired signatureFAILFAILFAILMissing signatureFAILFAILFAIL

Like this tool? Star us on GitHub. 

38ms IPv6 EDNS DNSSEC 

dns: 48

>
